build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -128,8 +158,11 @@ function getFiles(testType) {
       if (argv.files) {
         return files.concat(getFilesFromArgv());
       }
-      if (argv.saucelabs) {
-        return files.concat(testConfig.unitTestOnSaucePaths);
+      if (
+        argv.saucelabs ||
+        (isGithubActionsBuild() && (argv.safari || argv.firefox))
```

Comment:
I thought this was going to be removed? So it's just
`if (argv.saucelabs || isGithubActionsBuild())`

Reply:
Hmm. I added the extra clauses thinking we might run tests on Chrome on GH actions at some point. But since we already do so on Travis, I've simplified the logic as you suggest.